src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -179,6 +179,7 @@ const ActiveState = ({
                                             can_reply: getProp(item.permissions, 'can_reply', false),
                                             can_resolve: getProp(item.permissions, 'can_resolve', false),
                                         }}
+                                        shouldTruncate
```

Comment:
So it looks like shouldTruncate doesn't determine whether or not a comment is truncated it determines if truncation should be possible. shouldTruncate could be true but truncation may never happen because the messages aren't long enough right? If this is the case then It might be more intuitive to name this something along the lines of **allowTruncation**.

Reply:
Exactly! 💯 Great suggestion! I double checked in designs, and I've noticed they use different name for this `Collapse` so I think I'll go with full rename for the component.
But will definitely change this prop to `allowCollapsing`

Reply:
this is minor but `allow` isn't a common prefix we use for booleans. Would `canCollapse` suffice?